lib/browser.js

```diff
@@ -81,7 +82,8 @@ extend(KeenCore.utils, {
   deepExtend,
   listener,
   serializeForm,
-  timer
+  timer,
+  setOptOut
```

Comment:
because other tracking libs are exposing this method within a client instance, we need to move setOptOut to line :46 , so we'll be able to run something like:
```
client.setOptOut(true);
```
on the client instance